packages/tinacms/src/admin/api.ts

```diff
@@ -97,6 +97,7 @@ export class TinaAdminApi {
                     filename
                     extension
                   }
+                  _values
```

Comment:
While the PR is nicely small, you might agree that it's not very bandwidth-efficient.

What I could change here, is to introduce a `_sys.fields` attribute
in `packages/@tinacms/graphql/src/resolver/index.ts` instead:
an object that would have only the subset of fields.

The question is: what that subset should contain?

- everything but the one with `isBody: true` ?
- everything but those with `type: "rich-text"`?
- only those with simple types? (not `rich-text` and not `object`)

That is a design decision, and you probably in a better position than me to answer that.

Reply:
Yeah this is a good point, it has the potential to de-optimize that query, it might be best to provide a property on field definitions that would populate it to the `_sys.fields` property, as you noted. 

It's not so much a matter of the size of payload, but also if we can avoid providing the `_values` then we can skip retrieving the document entirely (basically the same thing we do with `_sys.title`, except it'd be a `JSON` type probably.

Reply:
Just thinking ahead a bit, If the user can specify which fields they want available to that list-view, maybe we should allow them to create new columns for a given field?
In that case, instead of using itemProps to control the label field, they'd be able to control the UI for any given column, as opposed to just the title-label.

<img width="707" alt="Screen Shot 2023-01-17 at 9 39 00 AM" src="https://user-images.githubusercontent.com/3323181/212913668-2a9c3e43-03bf-4590-a3e4-0358c03a10a4.png">

(Another common example might be including a custom "author" field in this list).